### ISSUE TITLE
Darksky Watering Data needs today's precip

### ISSUE THREAD

**Issue by PeteBa** [OpenSprinkler/OpenSprinkler-Weather#29]
We should include today's rainfall as well as yesterday's rainfall in `getDarkSkyWateringData()`.

**Comment by rmloeb** [comment]
I got weather-server operational under Linux 18.04 with help from salbahra. (Thank you.)

Been through a series of strange results, itemized in help desk ticket #7949.  May briefly have had it working, sort of, using DarkSky for data.  Seem to have received one forecast that was close to correct.  However, last successful weather call is empty, current temperature isn't updating, forecast is obsolete, and Zimmerman calculation is screwy.  I've been experimenting with calling weather-server directly with various Zimmerman parameters and the results don't make sense, i.e., no matter what parameters I enter, the result is always the same.  (And it's unclear how to enter a daily expected rainfall amount of.2 inches.)  Is there a log function I can turn on in weather-server?

**Comment by rmloeb** [comment]
How do I enter a west (negative) latitude in a manual call to weather-server?  If I enter 39,-104, I get Unable to revolve location.  Also, can I enter a more precise location, e.g., 39.4,-104.7?

**Comment by salbahra** [comment]
You can pass a full coordinate set which will match the GPS regular expression allowing it to bypass location resolution. The following regular expression is used:

```
/^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?),\s*[-+]?(180(\.0+)?|((1[0-7]\d)|([1-9]?\d))(\.\d+)?)$/
```

**Comment by PeteBa** [comment]
All, this issue is very specific to the detailed calculation of the DarkSky implementation. Can I ask that we take "usage" discussions to the forum or raise another issue for a new concern. Not looking to increase the frustration levels just trying to keep the specifics of the Original Post from being diluted..

**Comment by rmloeb** [comment]
My apologies.  I actually meant to open a new issue.  Been wrestling with this for several hours because I'm desperate to get Zimmerman working before I have to go out of town.  I do very much appreciate the hard work that is being done!

**Comment by PeteBa** [comment]
@rmloeb if you open a new issue then I'll be happy to help out. But please note that DarkSky is very experimental and not quite ready for prime-time.

**Comment by rmloeb** [comment]
Thanks. For now I'll switch over to OWM.

**Comment by Derpthemeus** [comment]
@PeteBa What is the motivation behind this change?

**Comment by Derpthemeus** [comment]
Resolved in #38
